src/io/psse.jl

```diff
@@ -501,25 +552,30 @@ end
     parse_psse(pti_data)
 
 Converts PSS(R)E-style data parsed from a PTI raw file, passed by `pti_data`
-into a format suitable for use internally in PowerModels.
+into a format suitable for use internally in PowerModels. Imports all remaining
+data from the PTI file if `import_all` is true (Default: false).
 """
-function parse_psse(pti_data::Dict)::Dict
+function parse_psse(pti_data::Dict; import_all=false)::Dict
     pm_data = Dict{String,Any}()
 
     pm_data["multinetwork"] = false
     pm_data["per_unit"] = false
     pm_data["source_type"] = "pti"
     pm_data["source_version"] = VersionNumber("$(pti_data["CASE IDENTIFICATION"][1]["REV"])")
-    pm_data["baseMVA"] = pti_data["CASE IDENTIFICATION"][1]["SBASE"]
-    pm_data["name"] = pti_data["CASE IDENTIFICATION"][1]["NAME"]
+    pm_data["baseMVA"] = pop!(pti_data["CASE IDENTIFICATION"][1], "SBASE")
+    pm_data["name"] = pop!(pti_data["CASE IDENTIFICATION"][1], "NAME")
 
-    psse2pm_bus!(pm_data, pti_data)
-    psse2pm_load!(pm_data, pti_data)
-    psse2pm_shunt!(pm_data, pti_data)
-    psse2pm_generator!(pm_data, pti_data)
-    psse2pm_branch!(pm_data, pti_data)
-    psse2pm_transformer!(pm_data, pti_data)
-    psse2pm_dcline!(pm_data, pti_data)
+    if import_all
+        import_remaining!(pm_data, pti_data["CASE IDENTIFICATION"][1]; exclude=["REV"])
```

Comment:
why exclude rev?

Reply:
rev is already used above, but pop! didn't work correctly inside the string

Reply:
To be consistent with the `pop!` behavior with the other parameters consider saving `pop!(pti_data["CASE IDENTIFICATION"][1], "REV")` to a temp variable and then turning it into a version number.